legacy-packages/title-block/KaizenDraft/TitleBlock/TitleBlock.tsx

```diff
@@ -56,7 +57,7 @@ const meetsCompactThreshold = () =>
   (window.scrollY || window.pageYOffset) >= COMPACT_NAVIGATION_SCROLL_THRESHOLD
 
 class TitleBlock extends React.Component<Props, State> {
-  static defaultProps = {
+  static defaultProps: Pick<Props, "textDirection"> = {
```

Comment:
Typescript doesn't enforce types on defaultProps - this change ensures `textDirection` is using the same type for Props and default props